samtranslator/internal/schema_source/aws_serverless_function.py

```diff
@@ -278,6 +278,7 @@ class ApiEventProperties(BaseModel):
     RequestModel: Optional[RequestModel] = apieventproperties("RequestModel")
     RequestParameters: Optional[RequestModelProperty] = apieventproperties("RequestParameters")
     RestApiId: Optional[Union[str, Ref]] = apieventproperties("RestApiId")
+    TimeoutInMillis: Optional[SamIntrinsicable[int]]  # TODO: add doc
```

Comment:
Why `SamIntrinsicable` ? I think we use `PassThroughProp` now

Reply:
Yep we can use `PassThroughProp` here